pkg/odo/cli/component/common_push.go

```diff
@@ -44,6 +45,19 @@ func NewCommonPushOptions() *CommonPushOptions {
 	}
 }
 
+func (cpo *CommonPushOptions) InitConfigFromContext() error {
```

Comment:
please add some comments to the function

Reply:
added